WeWatch/WeWatch/Splash/SplashViewModel.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SplashViewModel.swift
+//  WeWatch
+//
+//  Created by Anton on 18/12/2024.
+//
+
+import Foundation
+
+internal final class SplashViewModel: ObservableObject {
+    
+    internal enum JwtViewError: Error {
+        case typeChangeError
+        case invalidExpirationClaim
+        case dataError
+    }
+    
+    internal enum KeychainKey {
+        static let token: String = "token"
+    }
+    
+    internal enum JWTClaim {
+        static let expiration = "exp"
+    }
+    
+    @Published private var token: String?
+    @Published private var errorMessage: String?
```

Comment:
If you can use private here or you don't need these properties work with views, don't make it @Published. Use just regular properties like `private var` etc. @Published is needed whenever you need to view react on changes.